README.md

```diff
@@ -1 +1,145 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [your_project_name] <a name="about-project"></a>
+
+this project was done with HTML, CSS AND JAVASCRIPT(ES6).
+it's a book list project that allows you to add book both in the ui and the localstorage and also gives you chance to delete from both
+
+**[your_project__name]** is a...
+
+## 🛠 Built With <a name="built-with">HTML, CSS AND JAVASCRIPT</a>
+
+-HTML
+-CSS
+-JAVASCRIPT
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+  ALLOWS YOU TO ADD BOOK
+- **[key_feature_2]**
+  ALLOWS YOU TO DELETE BOOK
+- **[key_feature_3]**
+  SINGLE WEBPAGE
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://dmambo.github.io/Awesome-bookes6/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To use this project.
+First you need to clone the respository into your local machine.
+
+### Prerequisites
+
+In order to run this project you need:
+
+html, css and javascript.
+
+### Setup
+
+Clone this repository to your desired folder:
+cd my-folder
+git clone git@github.com:myaccount/my-project.git
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Dmambo)
+- Twitter: [@twitterhandle](https://twitter.com/kingibro345)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Ibrahim mohammed diaby)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/Rikol137)
+- Twitter: [@twitterhandle](https://twitter.com/RikolK)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/Kelvin Kimwetich)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Your link to your license file is broken 😨 Because you didn't add a license file 😄 Kindly add a license file using these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) 👍